src/test/java/com/artipie/asto/RxFileTest.java

```diff
@@ -33,18 +33,14 @@
 import java.util.Arrays;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 /**
  * Test case for {@link RxFile}.
  * @since 0.11.1
  */
-public class RxFileTest {
+final class RxFileTest {
 
-    /**
-     * Test that {@link RxFile#flow()} works.
```

Comment:
@g4s8 why we removed methods javadoc, but left javadoc on classes?

Reply:
@iakunin because the checkstyle rule requires javadoc for classes but doesn't require for methods. I think the method name is clear enough here to skip javadoc.

Reply:
@g4s8 got it, thanks